backend/src/controllers/reports/Tab_10_rpt_CA_MultiYrStats_ConsultingServicesbyPortfolio.js

```diff
@@ -18,7 +18,6 @@ controller.getReport = getReport;
  * @returns {object}
  */
 controller.Tab_10_rpt_CA_MultiYrStats_ConsultingServicesbyPortfolio = async (request, reply) => {
-  controller.userRequires(request, "PMO-Reports-Capability", reply);
```

Comment:
since this is not the generic controller, the route for this report needs the config.role added, look at the route for the generic controller for reports.   If i missed it, please point me to where it was updated

Reply:
luckily this controller is useless, and was already replaced by the generic controller. I've deleted it after printing a report to test the model.